test/upload-test-images.sh

```diff
@@ -17,14 +17,10 @@
 set -o errexit
 
 : ${1:?"Pass the directories with the test images as arguments"}
-: ${DOCKER_REPO_OVERRIDE:?"You must set 'DOCKER_REPO_OVERRIDE', see DEVELOPMENT.md"}
+: ${KO_DOCKER_REPO:?"You must set 'KO_DOCKER_REPO', see DEVELOPMENT.md"}
 
-DOCKER_FILES="$(find $@ -name Dockerfile)"
-: ${DOCKER_FILES:?"No subdirectories with Dockerfile files found in $@"}
+IMAGE_DIRS="$(find $@ -depth 1 -type d)"
```

Comment:
Instead of `$@`, use the location relative to this script, which is `$(dirname $0)/test_images`.

Reply:
Done